apps/openchallenges/challenge-service/src/main/resources/db/challenge_data_edam.csv

```diff
@@ -0,0 +1,304 @@
+"id","challenge_id","edam_id"
+"1","1","topic_3120"
```

Comment:
This table should only reference foreign keys, which is not the case of `edam_id`. A related question is: should we make `edam_id` the primary key in the table `edam_terms.csv`?

Reply:
Yes, using `edam_id` as PK makes sense to me.

Reply:
DB-related question: now that `edam_id` is the PK, should I go ahead and remove `id` from the table?

Reply:
Yes